packages/manager/apps/pci-ai-tools/src/data/hooks/ai/capabilities/useGetQpuFlavors.hook.tsx

```diff
@@ -0,0 +1,24 @@
+import { QueryObserverOptions, UseQueryResult } from '@tanstack/react-query';
+import { getQpuFlavors } from '@/data/api/ai/capabilities/capabilities.api';
+import { useQueryImmediateRefetch } from '@/hooks/useImmediateRefetch';
+import quantum from '@/types/Quantum';
+
+export function useGetQpuFlavors(
+  projectId: string,
+  region: string,
+  options: Omit<QueryObserverOptions, 'queryKey'> = {},
+) {
+  const queryKey = [
+    projectId,
+    'quantum',
+    'capabilities',
+    'region',
+    region,
+    'qpuFlavor',
```

Comment:
Should match the API key: `/cloud/project/${projectId}/quantum/capabilities/region/${region}/qpu`, 
```suggestion
    'qpu',
```